deploy.sh

```
@@ -0,0 +1,99 @@
+    echo "  - 8080 포트 점유 프로세스 (PID: $TARGET_PID) 종료 시작" | tee -a "$DEPLOY_LOG"
+    kill -9 $TARGET_PID
+    echo "  - 8080 포트 점유 프로세스 (PID: $TARGET_PID) 종료 완료" | tee -a "$DEPLOY_LOG"
+fi
+echo "2. 8080 포트 점유 프로세스 종료 끝" | tee -a "$DEPLOY_LOG"
+
+
+# 3. 최신 소스 코드 가져오기 (Git Clone or Pull)
+echo "3. 최신 소스 코드 가져오기 시작" | tee -a "$DEPLOY_LOG"
+if [ -d "$PROJECT_DIR" ]; then
+    echo "  - git pull 수행 시작" | tee -a "$DEPLOY_LOG"
+    cd "$PROJECT_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)